src/main/java/bio/terra/cli/command/resource/update/BqDataset.java

```diff
@@ -36,7 +39,8 @@ protected void execute() {
     // all update parameters are optional, but make sure at least one is specified
     if (!resourceUpdateOptions.isDefined()
         && !bqDatasetLifetimeOptions.isDefined()
-        && !bqDatasetNewIds.isDefined()) {
+        && !bqDatasetNewIds.isDefined()
+        && newCloningInstructionsOption.getCloning() == null) {
```

Comment:
It might be nice to make our own `isDefined()` method to move this null check into the class like the other mixins are doing.